.github/workflows/actions.yml

```diff
@@ -306,7 +307,7 @@ jobs:
         run: cmake --build $BUILD_PATH --config Release --target install
       - name: create archive
         if: matrix.artifact-suffix
-        run: cd $INSTALL_PATH && zip --symlinks -r $ARTIFACT_FILE SuperCollider # this assumes that we end up with the build in the folder SuperCollider
+        run: cd $INSTALL_PATH && hdiutil create -volname "$DISK_IMAGE_NAME" -srcfolder SuperCollider -ov -format UDBZ $ARTIFACT_FILE # this assumes that we end up with the build in the folder SuperCollider
```

Comment:
i think -ov is probably not needed here, the dmg shouldn't exist so there is nothing to overwrite, or am i missing something?